.travis.yml

```diff
@@ -26,30 +30,30 @@ git:
 env:
   global:
     - ISSUES_REPO_URI=webcompat/webcompat-tests/issues
-    # grab latest 4.x release, which is LTS for now
-    - TRAVIS_NODE_VERSION="4"
 
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
 
 install:
-  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install node
+  - node --version
   - travis_retry npm install
   - travis_retry npm run pip
   - travis_retry npm run config
   - which firefox
   - java -version
-  - firefox --version
   # lint python
   - pep8 --ignore=E402 webcompat/ tests/ config/secrets.py.example
   - python run.py -t &
 
 before_script:
   - "sleep 2"
   # lint JS in default eslint task
-  - npm run lint
-  - npm run build
+  - npm run lint --force
```

Comment:
Any reason you added `--force` here? I think ideally we want lint to fail a build, that way errors don't get into master.

Reply:
Hm, I added force, because it would't run these two scripts and it said the reason was because it didn't have `--force` in them. Since we are using a deprecated image, it seems to be required.

Indeed, it defeats the purpose if we force it... I'll look into it.

Reply:
Hmmmmm, interesting.

Reply:
Can you paste the error message that it gives without `--force`?